app/assets/images/rails.png

```diff
@@ -0,0 +1,673 @@
+								<input type="email" id="form_user_email"  name="form_user_email" maxlength="255" placeholder="Email" /><span id="check-e"></span>
+								<div class="user_register_terms"><!-- -->
+								</div>									
+								<button type="submit" class="button_register" name="btn-save" id="btn-submit3">Request New Password</button>
+							</form>
+						</div>
+					</div>
+				</div>						
+			</section>
+		</div>		
+		<script src="/js/scripts3.min.js"></script>
+		<script>
+
+
+		$(".image_download_button").click(function() { 
+ 			$(".modal_trigger_reg").click();
+		});			
+		$(".image_like_button").click(function() { 
+ 			$(".modal_trigger_reg").click();
+		});
+		$(".image_download_button").click(function() { 
+ 			$(".modal_trigger_reg").click();
+		});
+		$(".mob_reg").click(function() { 
+ 			$(".modal_trigger_reg").click();
+		});
+		$(".mob_log").click(function() { 
+ 			$(".modal_trigger_log").click();
+		});			
+				</script>
+
+		
+<script src="//m.servedby-buysellads.com/monetization.js" type="text/javascript"></script>
+<script>
+(function(){
+  if(typeof _bsa !== 'undefined' && _bsa) {
+    _bsa.init('fancybar', 'CK7I45QL', 'placement:imgbincom');
+  }
+})();
+</script>
+<script src="/absafe/bubble.php"></script>
+		<script type="text/javascript">
+			function abdd() {$("#adb-enabled").show(), $("#adb-not-enabled").hide(), $("#abf_center").html("<iframe src='/absafe/bsa_cpc_small.php?a=c' id='ablc' marginwidth='0' align='center' marginheight='0' hspace='0' vspace='0' frameborder='0' scrolling='no' allowtransparency='true' width='100%' height='110' style='width: 100%; height: 110px;'></iframe>"),$("#abf_left").html("<iframe src='/absafe/bsa_cpc_small.php' id='abll' marginwidth='0' align='center' marginheight='0' hspace='0' vspace='0' frameborder='0' scrolling='no' allowtransparency='true' width='100%' height='110' style='width: 100%; height: 110px;'></iframe>")}
+			function abnd(){$("#adb-enabled").hide(), $("#adb-not-enabled").show()}$(function(){}), "undefined" == typeof fAdBlock ? abdd() : (fAdBlock.setOption({debug: !1}), fAdBlock.onDetected(abdd).onNotDetected(abnd));
+			
+		</script>	
+	
+ <!-- Histats.com  (div with counter) --><div id="histats_counter" style="display:none"></div>
+<!-- Histats.com  START  (aync)-->
+<script type="text/javascript">var _Hasync= _Hasync|| [];
+_Hasync.push(['Histats.start', '1,4255402,4,511,95,18,00000000']);
+_Hasync.push(['Histats.fasi', '1']);
+_Hasync.push(['Histats.track_hits', '']);
+(function() {
+var hs = document.createElement('script'); hs.type = 'text/javascript'; hs.async = true;
+hs.src = ('//s10.histats.com/js15_as.js');
+(document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(hs);
+})();</script>
+<noscript><a href="/" target="_blank"><img  src="//sstatic1.histats.com/0.gif?4255402&101" alt="hit counter script" border="0"></a></noscript>
+<!-- Histats.com  END  -->	
+	
+    </body>     
+</html>

```

Comment:
@shaniajubelag Is this file being used somewhere? Please verify.
If not, please remove.